modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -186,32 +186,13 @@ public SnapshotReader getReader() {
 
     /**
      * Prepares the {@link MvPartitionStorage} for a full rebalance.
-     *
-     * <p>Recreates {@link MvPartitionStorage} and sets the last applied index to {@link TableManager#FULL_RABALANCING_STARTED} so that
-     * when the node is restarted, we can understand that the full rebalance has not completed, and we need to clean up the storage from
-     * garbage.
      */
-    private CompletableFuture<?> prepareMvPartitionStorageForRebalance(Executor executor) {
+    private CompletableFuture<?> prepareMvPartitionStorageForRebalance() {
         if (canceled) {
             return completedFuture(null);
         }
 
-        return partitionSnapshotStorage.partition().reCreateMvPartitionStorage()
-                .thenComposeAsync(mvPartitionStorage -> {
-                    if (canceled) {
-                        return completedFuture(null);
-                    }
-
-                    mvPartitionStorage.runConsistently(() -> {
-                        mvPartitionStorage.lastApplied(FULL_RABALANCING_STARTED, 0);
-
-                        return null;
-                    });
-
-                    LOG.info("Copier prepared multi-versioned storage for the partition [partId={}, tableId={}]", partId(), tableId());
-
-                    return completedFuture(null);
-                }, executor);
+        return partitionSnapshotStorage.partition().reCreateMvPartitionStorage();
```

Comment:
The previous version of the code logged when it prepared the MV storage. Here, the logging is removed. Was this made on purpose, or should we still log it?